modeltrans/fields.py

```diff
@@ -88,13 +88,20 @@ def db_type(self, connection):
         return None
 
     def get_instance_fallback_chain(self, instance, language):
+        """
+        Return the fallback chain for the instance.
+
+        Most of the time, it is just the configured fallback chain, but if the per-record-fallback feature
+        is used, the value of the field is added (if not None).
+        """
         default = get_fallback_chain(language)
 
         i18n_field = instance._meta.get_field("i18n")
         if i18n_field.fallback_language_field:
             record_fallback_language = get_instance_field_value(
                 instance, i18n_field.fallback_language_field
             )
+
             if record_fallback_language:
                 return [record_fallback_language] + [default]
```

Comment:
@jieter hier loopt die nog vast omdat dit: [nl] + [(en,)] is bv.
Default is een tuple - daar kan build localized fieldname niks mee (foutmelding: replace is not a property of a tuple)

Ik kan het lokaal oplossen door hier boven:
default = [get_fallback_chain(language)[0]] 
te zetten, i.e. een list met een entry van te maken.

Reply:
Ah, deze!

Je voorstel is niet correct als de chain meerdere elementen heeft natuurlijk.